modules/microshift-ki-cni-iptables-deleted.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * microshift_support/microshift-version.adoc
+
+[id="microshift-ki-cni-iptables-deleted_{context}"]
+= Reloading the firewall deletes iptable rules
+
+OVN-Kubernetes handles incoming NodePort traffic by using iptable rules. When you reload firewall rules with the `firewall-cmd --reload` command, the iptable rules are deleted. This stops the NodePort service traffic and any other host traffic that uses iptable rules.
+//Q: Is firewall reload something that happens automatically and continuously? When does it happen?
+[id="microshift-ki-cni-iptables-deleted-workaround_{context}"]
+== Restarting the DaemonSet pod as a workaround
+To troubleshoot this issue, restart the ovnkube DaemonSet pod. Restarting the pod will trigger a reconciliation of the iptable rules.
```

Comment:
Is the pod named _ovnkube_? 
If not, s/ovnkube DaemonSet pod/ ovnkube daemon set pod
If so, s/ovnkube DaemonSet pod/`ovnkube` DaemonSet pod

Reply:
No, each pod will have a suffix that isn't predictable.